### PR TITLE
Add prop type node to Slider, CheckboxToggle and ProgressBar label

### PR DESCRIPTION
**Pull request by PeterSiegl**

### CODE REVIEW

File: src/components/CheckboxToggle/CheckboxToggle.js [hunk 1]
```diff
@@ -73,7 +73,10 @@ CheckboxToggle.propTypes = {
   /**
    * Checkbox label
    */
-  label: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([
+    PropTypes.string,
```
Comment (quadreex) [inline src/components/CheckboxToggle/CheckboxToggle.js]: If I am not mistaken `PropTypes.node.isRequired` can be used here directly, as it is compatible with strings also.
`PropTypes.node` - is the most generic type.

**Comment by quadreex** [comment]
@PeterSiegl Could you please also change the type of `label` prop for `Pill` component?

File: src/components/CheckboxToggle/CheckboxToggle.js [hunk 2]
```diff
@@ -73,7 +73,10 @@ CheckboxToggle.propTypes = {
...
+    PropTypes.node,
+  ]).isRequired,
```
Comment (fspoettel) [inline src/components/CheckboxToggle/CheckboxToggle.js]: Shouldn't `statusLabels` also be changed to keep this consistent?

**Comment by fspoettel** [comment]
@quadreex Changing this for `Pill` is potentially hazardous to the expected behavior specified in the LDS. It relies on truncation classes that only work with text (and probably `inline`) content, anything else will break that behavior.

File: src/components/CheckboxToggle/CheckboxToggle.js (hunk 1 above)
Comment (PeterSiegl) [reply src/components/CheckboxToggle/CheckboxToggle.js]: Agreed, I'll change other occurrences of this pattern too.

**Comment by quadreex** [comment]
@fspoettel I already use inline components with Pill. And I get the warning. I think that decision should be made by the user of a component and not the component itself about allowing to use different labels.

**Comment by fspoettel** [comment]
@quadreex I agree with this for the most part. We still should stick to the underlying LDS implementation as close as possible and [the docs](https://lightningdesignsystem.com/components/pills/) do not indicate support for anything but a string as pill content. May I ask what your use case is here? We may as well make an exception here and implement the propType if the usecase is reasonable

**Comment by quadreex** [comment]
@fspoettel @PeterSiegl Example where I use markup inside Pill:
https://github.com/propertybase/pba-react/blob/1551e1979258a30cabdab27b95a0674ff7c1b134/app/DripCampaigns/components/InputField/components/ContactDateFieldInput.js#L42

**Comment by fspoettel** [comment]
@quadreex Thanks! Let's make the change in that case.

**Comment by PeterSiegl** [comment]
@quadreex @fspoettel I adjusted the label prop type for pills. can I merge this?

**Pull request merged**